sql/core/src/main/scala/org/apache/spark/sql/execution/command/tables.scala

```diff
@@ -389,6 +389,50 @@ case class TruncateTableCommand(
 }
 
 /**
+ * A command to repair a table by discovery all the partitions in the directory.
+ *
+ * The syntax of this command is:
+ * {{{
+ *   MSCK REPAIR TABLE table_name;
+ * }}}
+ *
+ * This command is the same as AlterTableRecoverPartitions
+ */
+case class RepairTableCommand(tableName: TableIdentifier) extends RunnableCommand {
```

Comment:
Why do we have the command? It is exactly the same as the `AlterTableRecoverPartitionsCommand`, except for the wording of the errors.